### ISSUE TITLE
Sorting by month is alphabetical, not calendrical

### ISSUE THREAD

**Issue by bdice** [FlamingTempura/bibtex-tidy#32]
Hi! I recently discovered this tool and I'm extremely happy with it. I noticed a bit of unexpected behavior when sorting by year/month/day. The months are being sorted alphabetically and not by the calendar. Strangely, this alphabetical order still happens if I specify the months as `10`, `3`, and `4` instead of `oct`, `mar`, and `apr`. I tried to find the place in the source code where sorting occurs but didn't see it from a quick search. If someone can give me a pointer on where this code is located, I could try to make a PR. Also happy to review a PR if someone else makes it.

```bibtex
@book{impossible2,
  title = {The other impossible book},
  author = {Stefan Sweig},
  year = 1942,
  month = oct,
  publisher = {Dead Poet Society}
}
@book{impossible,
  title = {The impossible book},
  author = {Stefan Sweig},
  year = 1942,
  month = mar,
  publisher = {Dead Poet Society}
}
@book{sweig42,
  title = {The impossible book},
  author = {Stefa{n} Sweig},
  year = 1942,
  month = apr,
  publisher = {Dead Poet Society}
}
```

This command shows the sorting flags I'm using. You can flip `-month` to `month` to swap between A-Z months and Z-A months.
```bash
bibtex-tidy --curly --numeric --space=2 --align=0 --sort=-year,-month,-day YOUR_FILE.bib
```

**Comment by FlamingTempura** [comment]
This has been fixed in a83e4ff

The above example will now be correctly sorted in month order:

```bibtex
@book{impossible,
	title        = {The impossible book},
	author       = {Stefan Sweig},
	year         = 1942,
	month        = mar,
	publisher    = {Dead Poet Society}
}
@book{sweig42,
	title        = {The impossible book},
	author       = {Stefa{n} Sweig},
	year         = 1942,
	month        = apr,
	publisher    = {Dead Poet Society}
}
@book{impossible2,
	title        = {The other impossible book},
	author       = {Stefan Sweig},
	year         = 1942,
	month        = oct,
	publisher    = {Dead Poet Society}
}
```

**Comment by bdice** [comment]
@FlamingTempura Nice work! Thank you! 🚀
